src/guides/v2.4/release-notes/commerce-2-4-1.md

```diff
@@ -208,48 +208,74 @@ We have fixed hundreds of issues in the Magento 2.4.1 core code.
 
 <!--- MC-33788-->
 
-*  Upgrade no longer results in the sudden failure of the Galera cluster. Previously, the Galera cluster exited abruptly after re-indexing immediately after upgrade. During Magento upgrade, index tables are altered and the engine is changed from MEMORY to InnoDB. At this point, the content of these tables becomes out-of-sync between the nodes of the Galera cluster. [GitHub-25334](https://github.com/magento/magento2/issues/25334)
+*  Upgrade no longer results in the sudden failure of the Galera cluster. Previously, the Galera cluster exited abruptly after re-indexing immediately after upgrade. During Magento upgrade, index tables are altered, and the engine is changed from `MEMORY` to `InnoDB`. At this point, the content of these tables becomes out-of-sync between the nodes of the Galera cluster. [GitHub-25334](https://github.com/magento/magento2/issues/25334)
+
+<!--- MC-34254-->
+
+*  Disabling the PageBuilder module no longer affects the rendering of the product page.  Previously, custom layouts on the product page disappeared when the module was disabled, and Magento displayed a blank page.
 
 <!--- ENGCOM-7219-->
 
+*  You can now use `bin/magento sampledata:deploy` to deploy sample data as expected after installing Magento using Composer. Previously, Magento threw this error: `Git installations must deploy sample data from GitHub; see https://devdocs.magento.com/guides/v2.3/install-gde/install/sample-data-after-clone.html for more information`. _Fix submitted by Andrii Beziazychnyi in pull request [27481](https://github.com/magento/magento2/pull/27481)_. [GitHub-19481](https://github.com/magento/magento2/issues/19481)
+
 <!--- ENGCOM-7459-->
 
-<!--- ENGCOM-7265-->
+*  Storefront performance has improved by eliminating the unnecessary loading  of the `Datepicker` component. _Fix submitted by Mateusz Krzeszowiak in pull request [27860](https://github.com/magento/magento2/pull/27860)_. [GitHub-28823](https://github.com/magento/magento2/issues/28823)
 
-<!--- ENGCOM-7508-->
+<!--- ENGCOM-7265-->
 
-<!--- ENGCOM-7800-->
+*  Executing `php bin/magento s:up` now completes as expected. Previously, Magento displayed printed array content for caches. _Fix submitted by Sathish Subramanian in pull request [27567](https://github.com/magento/magento2/pull/27567)_. [GitHub-27091](https://github.com/magento/magento2/issues/27091)
 
 <!--- ENGCOM-7629-->
 
-<!--- ENGCOM-7776-->
+*  Running `./bin/magento config:show vendor_module/general/value` now returns `0` or '' (an empty string) as expected. Previously, this command returned `Configuration for path: "vendor_module/general/value" doesn't exist`. _Fix submitted by Vadim Malesh in pull request [28549](https://github.com/magento/magento2/pull/28549)_. [GitHub-23290](https://github.com/magento/magento2/issues/23290)
```

Comment:
Is already mentioned a couple of paragraphs above, best to merge it.

Reply:
merged. thanks. I need o watch out for fixes that tickets in both the ENGCOM and MC projects.